src/ui/osx/TogglDesktop/test2/UnsupportedNotice.m

```diff
@@ -0,0 +1,52 @@
+//
+//  UnsupportedNotice.m
+//  TogglDesktop
+//
+//  Created by Indrek Vändrik on 23/11/2018.
+//  Copyright © 2018 Alari. All rights reserved.
+//
+
+#import "UnsupportedNotice.h"
+
+@implementation UnsupportedNotice
+
++ (instancetype)sharedInstance
+{
+	static UnsupportedNotice *instance;
+	static dispatch_once_t onceToken;
+
+	dispatch_once(&onceToken, ^{
+					  instance = [[UnsupportedNotice alloc] init];
+				  });
+	return instance;
+}
+
+- (void)showNotice
+{
+	NSDate *today = [NSDate date];
+	NSDate *deadline = [NSDate dateWithString:@"2019-01-01 00:00:00 +0000"];
+	NSComparisonResult result = [today compare:deadline];
+
+	NSString *title = [NSString stringWithFormat:@"Mac OS X version not supported"];
+	NSString *text = @"";
+
+	if (result == NSOrderedAscending)
+	{
+		text = @"Toggl Desktop will stop working with your current version of OS X from the 1st of January 2019.\n\n";
```

Comment:
Off space ⚒

Reply:
Please elaborate what exactly do you mean by "Off space"?

Reply:
Ah, I just double check - It's good for now 👨‍💻